#!/usr/bin/env python
PACKAGE = "iri_finger_pointing"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("select_point",   bool_t,   0, "Checkbox to select point",  False)

exit(gen.generate(PACKAGE, "iri_finger_pointing", "Reconfig_params"))
